static/js/browser2/statvar_group_node.tsx

```diff
@@ -226,6 +226,17 @@ export class ChildStatVarGroupSection extends React.Component<
   ChildStatVarGroupSectionPropType
 > {
   render(): JSX.Element {
+    const childStatVarGroups = this.props.data[this.props.statVarGroupId]
```

Comment:
since there is only one such node, it may be more efficient to do a linear scan to find the node, and move it up top.

Reply:
done